subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenBomResolveIntegrationTest.groovy

```diff
@@ -69,6 +70,7 @@ class MavenBomResolveIntegrationTest extends AbstractHttpDependencyResolutionTes
         }
     }
 
+    @Ignore("This is no longer true, since a dependency on a BOM is by default a dependency on a library")
```

Comment:
Should this test be inverted to become: "Cannot import a bom transitively" ?
Or simply removed as we have no final say on this feature

Reply:
Removed test